about/bibliography/index.md

```diff
@@ -52,7 +52,7 @@ This online bibliography contains a list of publications by ADS staff and studie
 <ul>
 <li>Accomazzi, A., M. Kurtz, E. Henneken, C. Grant, D. Thompson, R. Chyla, S. McDonald, S. Blanco-Cuaresma, G. Shapurian, T. Hostetler, M. Templeton, K. Lockhart, and K. Bukovi (2020), [Life after Classic: An Astronomer's Guide to the new ADS](https://ui.adsabs.harvard.edu/abs/2020AAS...23528705A/abstract), American Astronomical Society Meeting Abstracts, 52, 287.05.</li>
 <li>van de Sandt, S., L. Holm Nielsen, A. Ioannidis, A. Muench, E. Henneken, A. Accomazzi, C. Bigarella, J. Benito Gonzalez Lopez, and S. Dallmeier-Tiessen (2019), [Practice meets Principle: Tracking Software and Data Citations to Zenodo DOIs](https://ui.adsabs.harvard.edu/abs/2019arXiv191100295V/abstract), arXiv e-prints, arXiv:1911.00295.</li>
-<li>Blanco-Cuaresma, S., A. Accomazzi, M. J. Kurtz, E. A. Henneken, C. S. Grant, D. M. Thompson, R. Chyla, S. McDonald, G. Shapurian, T. W. Hostetler, M. R. Templeton, K. E. Lockhart, K. Bukovi, and N. Rapport (2019), [Fundamentals of Effective Cloud Management for the New NASA Astrophysics Data System](https://ui.adsabs.harvard.edu/abs/2019ASPC..523..353B/abstract), Astronomical Data Analysis Software and Systems XXVII, 523, 353.</li>
+<li>Blanco-Cuaresma, S., A. Accomazzi, M. J. Kurtz, E. A. Henneken, C. S. Grant, D. M. Thompson, R. Chyla, S. McDonald, G. Shapurian, T. W. Hostetler, M. R. Templeton, K. E. Lockhart, K. Bukovi, and N. Rapport (2019), [Fundamentals of Effective Cloud Management for the New Astrophysics Data System](https://ui.adsabs.harvard.edu/abs/2019ASPC..523..353B/abstract), Astronomical Data Analysis Software and Systems XXVII, 523, 353.</li>
```

Comment:
Hmm I suspect we should leave this one as-is, since it's the name of a published conference proceeding